src/topology/mod.rs

```diff
@@ -288,16 +282,14 @@ impl RunningTopology {
 
         // First pass to tell the sources to shut down.
         let mut source_shutdown_complete_futures = Vec::new();
-        // TODO: Once all Sources properly look for the ShutdownSignal, up this time limit to something
-        // more like 30-60 seconds.
 
         // Only log that we are waiting for shutdown if we are actually removing
         // sources.
         if !diff.sources.to_remove.is_empty() {
             info!("Waiting for up to 3 seconds for sources to finish shutting down");
```

Comment:
Do we need to change this message? I also think 30 seconds is plenty here.

Reply:
Indeed.